tests/Tests/800__ocm/800__ocm.robot

```diff
@@ -0,0 +1,35 @@
+*** Settings ***
+Library         SeleniumLibrary
+Resource        ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
+Resource        ../../Resources/Page/HybridCloudConsole/HCCLogin.robot
+Suite Setup     OCM Test Setup
+Suite Teardown  OCM Test Teardown
+
+*** Test Cases ***
+Verify User Can Access RHODS Documentation
+    [Tags]  ODS-1303
+    Click Button        //button[@data-ouia-component-id="Add-ons"]
+    Click Element       //img[@alt="Red Hat OpenShift Data Science"]
+    Wait Until Page Contains Element    //div[@class="pf-c-drawer__panel ocm-c-addons__drawer--panel-content"]
+    Page Should Contain Element     //a[@href="https://access.redhat.com/documentation/en-us/red_hat_openshift_data_science"]
+    Verify Documentation Is Accessible
+
+*** Keywords ***
+Verify Documentation Is Accessible
+    ${status}=    Check HTTP Status Code    https://access.redhat.com/documentation/en-us/red_hat_openshift_data_science
+    Run Keyword IF  ${status}!=200      FAIL
+    ...     Documentation Is Not Accessible
+
+OCM Test Setup
+    Set Library Search Order    SeleniumLibrary
```

Comment:
this is not going to work on stage clusters because the link you get from Switcher (line 29)  is not working on stage. You need to change it accordingly to make it work